.infrastructure/pvc.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+ name: pvc-data
+ namespace: todoapp
+spec:
+ volumeName: pv-data
+ accessModes:
+   - ReadWriteMany
+ resources:
+   requests:
+     storage: 1G
```

Comment:
The storage request should be specified as '1Gi' instead of '1G'. Kubernetes uses 'Gi' for gibibytes, which is the correct unit for storage requests.